transformBAM.sh

```diff
@@ -55,7 +55,8 @@ if [[ ! -e "$FILE_IN" ]];then
     logMsg "ERROR" "Input file does not exists: ($FILE_IN)"
 fi
 DIR_IN="$(pwd)"
-originalSize=$(samtools view -c -@ 4 "$DIR_IN/$FILE_IN")
+logMsg "INFO" "Calculating number of entries in original BAM."
+originalSize=$(samtools view -c -@ 8 "$DIR_IN/$FILE_IN")
```

Comment:
Can the number of additional threads become a parameter of transformBAM script? We might want to increase (or shrink) it depending on the size of the BAMs, and it's reused several times.

Reply:
I thought about this, but I kind of recall that samtools doesn't get much better when using more than 8 cores. If I'm wrong, we should definitely parametrize this.